modules/arm_plugin/src/arm_converter/arm_converter_pool.cpp

```diff
@@ -191,16 +192,34 @@ struct NEPoolingLayerQI final: public arm_compute::IFunction {
     std::unique_ptr<arm_compute::NEPoolingLayer> _pool;
 };
 template<> Converter::Conversion::Ptr Converter::Convert(const opset::AvgPool& node) {
-    arm_compute::PoolingLayerInfo pool_info;
-    FillLayerInfo(node, pool_info);
-    pool_info.pool_type       = arm_compute::PoolingType::AVG;
-    pool_info.exclude_padding = node.get_exclude_pad();
-    auto iInfoIt = node.get_rt_info().find("InputPrescaleInfo");
-    const arm_compute::QuantizationInfo* iInfo = iInfoIt == node.get_rt_info().end() ? nullptr :
-                                               &(iInfoIt->second.as<arm_compute::QuantizationInfo>());
-    auto qInfoIt = node.get_rt_info().find("QuantizationInfo");
-    const arm_compute::QuantizationInfo* qInfo = qInfoIt == node.get_rt_info().end() ? nullptr :
-                                               &(qInfoIt->second.as<arm_compute::QuantizationInfo>());
-    return MakeConversion<NEPoolingLayerQI>(node.input(0), node.output(0), pool_info, iInfo, qInfo);
+    if (node.get_input_shape(0).size() == 4) {
+        arm_compute::PoolingLayerInfo pool_info;
+        FillLayerInfo(node, pool_info);
+        pool_info.pool_type       = arm_compute::PoolingType::AVG;
+        pool_info.exclude_padding = node.get_exclude_pad();
+        auto iInfoIt = node.get_rt_info().find("InputPrescaleInfo");
+        const arm_compute::QuantizationInfo* iInfo = iInfoIt == node.get_rt_info().end() ? nullptr :
+                                                &(iInfoIt->second.as<arm_compute::QuantizationInfo>());
+        auto qInfoIt = node.get_rt_info().find("QuantizationInfo");
+        const arm_compute::QuantizationInfo* qInfo = qInfoIt == node.get_rt_info().end() ? nullptr :
+                                                &(qInfoIt->second.as<arm_compute::QuantizationInfo>());
+        return MakeConversion<NEPoolingLayerQI>(node.input(0), node.output(0), pool_info, iInfo, qInfo);
+    } else {
+        auto make = [&] (auto refFunction) {
```

Comment:
Could you please add here assertions that neither `InputPrescaleInfo` nor `QuantizationInfo` are provided for the node.

Also it would be necessary to check input shape at
https://github.com/openvinotoolkit/openvino_contrib/blob/cdaa9274680fe15cede13bb3d76209b663d13211/modules/arm_plugin/src/transformations/quantize_fusion.cpp#L259
and
https://github.com/openvinotoolkit/openvino_contrib/blob/cdaa9274680fe15cede13bb3d76209b663d13211/modules/arm_plugin/src/transformations/quantize_fusion.cpp#L397
to avoid conversion of unsuitable operations and creation of quantization information for the node